### PR TITLE
[Storage] wire ok http client in tests.

### PR DESCRIPTION
**Pull request by kasobol-msft**
To address https://github.com/Azure/azure-sdk-for-java/issues/21147

### CODE REVIEW

File: sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java [hunk 1]
```diff
@@ -33,6 +35,8 @@
     private TestEnvironment() {
         this.testMode = readTestModeFromEnvironment();
         this.serviceVersion = readServiceVersionFromEnvironment();
+        this.httpClientType = readHttpClientTypeFromEnvironment();
+        System.out.println(String.format("Tests will run with %s http client", this.httpClientType));
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java:39]: Can remove this once the PR is ready for merging

**Review by alzimmermsft — APPROVED** [review]

File: sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java (hunk 1 above)
Comment (kasobol-msft) [reply sdk/storage/azure-storage-common/src/test-shared/java/com/azure/storage/common/test/shared/TestEnvironment.java:39]: I left this on purpose (like other printouts in this class are).

**Pull request merged**